generate-manifest.py

```diff
@@ -53,6 +58,17 @@ def edit_manifest(data, arch, branch, runtime_version):
     data['branch'] = branch
     data['runtime-version'] = runtime_version
 
+    # This is nasty
+    gtk_patch = { 'type': 'patch', 'file': 'gtk3-egl-x11.patch' }
+    u = request.urlopen(FREEDESKTOP_MANIFEST_URL)
```

Comment:
I would have expected that we would want to track the version in org.gnome.Platform anyway, since that's built after (and should overwrite) the one in org.freedesktop.Platform.

Reply:
Oh, I didn't read the Phabricator comment carefully enough. There is no org.gnome.Platform version in 3.24.